channels/us.m3u

```diff
@@ -305,6 +305,9 @@ https://etlive-mediapackage-fastly.cbsaavideo.com/dvr/manifest.m3u8
 https://stream.swagit.com/live-edge/escambiacountyfl/live-1-a/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/fGdHIGy.png" group-title="Religious",ESNE TV
 https://zypelive-lh.akamaihd.net/i/default_1@710948/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://static.epg.best/us/ESPN.us.png" group-title="Sport",ESPN
+#EXTVLCOPT:http-referrer=https://espn-live.stream/stream/43572.html
+https://e10.cdn4.us/ingest666/espn_usa/f.m3u8
```

Comment:
The stream's not working at the moment. Tested in VLC player.
<img width="317" alt="Screenshot 2020-07-27 at 17 36 42" src="https://user-images.githubusercontent.com/7253922/88555078-fa52eb80-d02f-11ea-9b5c-ee79a29f68e1.png">

Reply:
Yes was working all weekend but not working now - I will close the pull until I find a working link again